# This is the AWS Client Configuration used by the AWS service client in the Node. If given the node will load the
# provided configuration when initializing the client.
aws_client_configuration:
    # Specifies where you want the client to communicate. Examples include us-east-1 or us-west-1. You must ensure that
    # the service you want to use has an endpoint in the region you configure.
    region: "us-west-2"
